### PR TITLE
2.0.0 Release

### PR DESCRIPTION
**Pull request by ryanwhowe**
# Description

Updates to support php 8.0

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

All internal unit tests have passed

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ and this project adheres to [Semantic Versioning].
 
 - The 2.x branch will be added to support php ^8.0
 
+## 2.0.0
```
Comment (AxeTools-PHP) [inline CHANGELOG.md]: We will need to include the date once the release date is set

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ and this project adheres to [Semantic Versioning].
...
+
+### Added
+
+- Initial release 2.0.0 ([#4])
+- Updated required php version to ^8.0
+- Added return type of `void` to `setCache()` method
```
Comment (AxeTools-PHP) [inline CHANGELOG.md]: Add the issue number to these changes

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,4 @@
-FROM php:7.0-fpm-stretch
-
-RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
```
Comment (AxeTools-PHP) [inline Dockerfile:3]: instead of waiting for this to happen maybe we should include the archive for bullseye to future proof the container building

**Review by AxeTools-PHP — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (ryanwhowe) [reply Dockerfile:3]: Unfortunately until a dist is placed into archive it is not in archive so adding this for bullseye is causing an error in the docker build process

**Review by AxeTools-PHP — APPROVED** [review]

**Pull request merged**